flaskbb/app.py

```diff
@@ -28,9 +29,10 @@
 # models
 from flaskbb.user.models import User, Guest
 # extensions
-from flaskbb.extensions import (db, login_manager, mail, cache, redis_store,
-                                debugtoolbar, alembic, themes, plugin_manager,
-                                babel, csrf, allows, limiter, celery, whooshee)
+from flaskbb.extensions import (alembic, allows, babel, cache, celery, csrf,
```

Comment:
Bit of noise here from when the pluggy instance was created here instead of on the application.

Reply:
but it's sorted alphabetically now :)